.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
```

Comment:
- [ ] One of the hardest things about being a Developer is naming things. We should always choose our words wisely, try always to use words that describe and are meaningful to the things you're doing.

Let's edit the branch name to something relevant, it could be an action or a feature. Example: `created_sidebar` or `index_sidebar_menu`.

- [ ] Being descriptive and meaningful also applies to our descriptions in our commit messages. Is not the same `fix linter issues` to `Fix html indentation in index file`. We should always describe in a few words our intents.

Reply:
- [ ] @bettercallsully The branch name is still not fixed yet, we need it to be descriptive as it was mentioned by the previous reviewer

Reply:
How do propose a descriptive branch name with 3 lines of code?!